src/TokenHandler/AccountManager.mo

```diff
@@ -74,21 +75,38 @@ module {
     /// Retrieves the allowed minimal deposit.
     public func minimum() : Nat = depositRegistry.minimum();
 
+    /// Sets the minimum deposit allowed.
+    /// Returns the new minimum, or `null` if minimum == prev_min or min <= fee.
+    /// Minimum must be greater than current fee.
+    public func setMinimum(min : Nat) : ?Nat {
```

Comment:
We could also return the previous minimum instead of null, making the return value Nat (without the option).

Not clear which one is better..